dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Controls/ExciterAvrPssOel.mo

```diff
@@ -0,0 +1,155 @@
+  Modelica.Blocks.Logical.Switch switch_OEL annotation(
+    Placement(visible = true, transformation(origin = {-26, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.RealToBoolean realToBoolean(threshold = 0) annotation(
+    Placement(visible = true, transformation(origin = {-64, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Add dIf(k1 = -1, k2 = 1) annotation(
+    Placement(visible = true, transformation(origin = {-124, 62}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain dIf_minus(k = -1) annotation(
+    Placement(visible = true, transformation(origin = {-88, 62}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Sources.Constant iFLimPu_const(k = IfLimPu) annotation(
+    Placement(visible = true, transformation(origin = {-172, 86}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain kMulDU(k = G) annotation(
+    Placement(visible = true, transformation(origin = {48, -38}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  OverExcitationLimitation OverExcitationLimitation1(f = f, r = r, y_start = -1) annotation(
+    Placement(visible = true, transformation(origin = {-158, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain efd_filtGain(k = 10) annotation(
+    Placement(visible = true, transformation(origin = {150, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Add efd_filtAdd(k1 = 1, k2 = -1) annotation(
+    Placement(visible = true, transformation(origin = {118, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.LimIntegrator efd_filtInt(k = 1, outMax = L2, outMin = 0, y_start = Efd0Pu) annotation(
+    Placement(visible = true, transformation(origin = {186, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput omegaPu annotation(
+    Placement(visible = true, transformation(origin = {-180, -112}, extent = {{-16, -16}, {16, 16}}, rotation = 0), iconTransformation(origin = {-120, -60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Math.Add dOmega(k1 = -1, k2 = 1) annotation(
+    Placement(visible = true, transformation(origin = {-126, -88}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Sources.Constant OmegaRefPu_const(k = 1) annotation(
+    Placement(visible = true, transformation(origin = {-174, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.LimIntegrator timer(k = 1, outMax = 99, outMin = L1, y_start = L1) annotation(
+    Placement(visible = true, transformation(origin = {-110, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.TransferFunction leadLag(a = {Tb, 1}, b = {Ta, 1}, initType = Modelica.Blocks.Types.Init.NoInit, x(fixed = true, start = Efd0Pu), y_start = Efd0Pu)  annotation(
```

Comment:
My OMEdit requires to put "each fixed" instead of "fixed" and "each start" instead of "start" for x.

Reply:
added each fixed and each start. resolved, see [e1000c7](https://github.com/dynawo/dynawo/pull/2394/commits/e1000c7adff25e271a2e2214bb7197d005b00633)

Reply:
> Also, does it work without the fixed ?

I would leave the fixed as is, because these are parameters that are dynamically calculated. 

https://mbe.modelica.university/behavior/equations/variables/ says the following:
> [...] Another, more obscure, use of the fixed attribute is for “computed parameters”. In rare cases where a parameter cannot be initialized explicitly, it is possible to provide a general equation for the parameter in an initial equation section. But in cases where the parameter is initialized in this way, the fixed attribute for the parameter variable must be set to false.